qa/automation/CareerPortal.Tests.Scenarios/2. Extended example/Test/CVforNotOpenPosition.feature

```diff
@@ -0,0 +1,122 @@
+Background: User went to the "Send us your CV" form at the down of the page 
+
+Feature: Send CV for not open job position  
+
+@Happy_pass
+Scenario: Send CV happy pass
```

Comment:
It is a good practice to use Tabs for indentation.

Reply:
O.k., I agree and I'll add tabs in my feature file.